win32ss/user/winsrv/consrv/shutdown.c

```diff
@@ -9,12 +9,84 @@
 /* INCLUDES *******************************************************************/
 
 #include "consrv.h"
+#include <psapi.h>
 
 #define NDEBUG
 #include <debug.h>
 
 /* FUNCTIONS ******************************************************************/
 
+extern ULONG_PTR LogonProcessId;
+
+static void 
+NotifyConsoleProcessForShutdown(IN PCSR_PROCESS CsrProcess,
+                                IN PCONSOLE_PROCESS_DATA ProcessData,
+                                IN ULONG Flags)
+{
+    DPRINT1("ConsoleClientShutdown(0x%p, 0x%x, %s) - Console process [0x%x, 0x%x]\n",
```

Comment:
Remove `, %s` too.

Reply:
Note: the %s was for the corresponding `FirstPhase ? "FirstPhase" : "LastPhase",` part of the old DPRINT.